src/lib/fileutils.ts

```diff
@@ -112,44 +112,156 @@ export const starterAzurePipelines = async (opts: {
+                return {
+                  script: generateYamlScript([
+                    `# --- From https://raw.githubusercontent.com/Microsoft/bedrock/master/gitops/azure-devops/release.sh`,
+                    `. build.sh --source-only`,
+                    ``,
+                    `# Initialization`,
+                    `verify_access_token`,
+                    `init`,
+                    `helm init`,
+                    `get_os`,
+                    ``,
+                    `# Fabrikate`,
+                    `get_fab_version`,
+                    `download_fab`,
+                    ``,
+                    `# Clone HLD repo`,
+                    `git_connect`,
+                    `# --- End Script`,
+                    ``,
+                    `# Update HLD`,
+                    `git checkout -b "DEPLOY/$(Build.Repository.Name)-${projectName}-$(Build.SourceBranchName)-$(Build.BuildNumber)"`,
+                    `../fab/fab set --subcomponent ${projectName} image.tag=$(Build.SourceBranchName)-$(Build.BuildNumber)`,
+                    `echo "GIT STATUS"`,
+                    `git status`,
+                    `echo "GIT ADD (git add -A)"`,
+                    `git add -A`,
+                    ``,
+                    `# Set git identity`,
+                    `git config user.email "admin@azuredevops.com"`,
+                    `git config user.name "Automated Account"`,
+                    ``,
+                    `# Commit changes`,
+                    `echo "GIT COMMIT"`,
+                    `git commit -m "Updating ${projectName} image tag to $(Build.SourceBranchName)-$(Build.BuildNumber)."`,
+                    ``,
+                    `# Git Push`,
+                    `git_push`,
+                    ``,
+                    `# Open PR via az repo cli`,
+                    `echo 'az extension add --name azure-devops'`,
+                    `az extension add --name azure-devops`,
+                    ``,
+                    `echo 'az devops login'`,
+                    `echo "$(PAT)" | az devops login`,
+                    ``,
+                    `echo 'az repos pr create --description "Updating ${projectName} to $(Build.SourceBranchName)-$(Build.BuildNumber)."'`,
+                    `az repos pr create --description "Updating ${projectName} to $(Build.SourceBranchName)-$(Build.BuildNumber)."`
+                  ]),
+                  displayName:
+                    "Download Fabrikate, Update HLD, Push changes, Open PR",
+                  env: {
+                    ACCESS_TOKEN_SECRET: "$(PAT)",
+                    REPO: "$(HLD_REPO)"
+                  }
+                };
+              })
+            ]
+          }
+        ]
+      }
     ]
   };
   // tslint:enable: object-literal-sort-keys
 
   const requiredPipelineVariables = [
     `'ACR_NAME' (name of your ACR)`,
+    `'HLD_REPO' (Repository for your HLD in AzDo. eg. 'dev.azure.com/bhnook/fabrikam/_git/hld')`,
```

Comment:
@sarath-p note this. Would it be possible to add these two variables to the command you are writing?